cmd/kubeadm/app/phases/kubeconfig/kubeconfig.go

```diff
@@ -433,6 +443,13 @@ func getKubeConfigSpecsBase(cfg *kubeadmapi.InitConfiguration) (map[string]*kube
 		kubeadmconstants.AdminKubeConfigFileName: {
 			APIServer:  controlPlaneEndpoint,
 			ClientName: "kubernetes-admin",
+			ClientCertAuth: &clientCertAuth{
+				Organizations: []string{kubeadmconstants.ClusterAdminsGroupAndClusterRoleBinding},
```

Comment:
Maybe we can load the Group name from the cluster instead of just using the constant.
If the user has already created `kubeadm:cluster-admins` CRB, but with a different custom Group name, kubeadm can respect the user's settings and use the custom Group with a warning.

Reply:
i was thinking about custom pre-created groups for a while and not sure how to pipe this to init and did not mention it in the KEP. right now this new code becomes integral in the init workflow and there is a bootstrapping requirement - super-admin.conf (with system:masters) must bootstrap the admin.conf to be a workable kubeconfig.

if the user skips the super-admin kubeconfig phase, they must provide the file out of band- e.g. by using external CA cert/key, otherwise there will be no power in the cluster to create the "kubeadm:cluster-admins" CRB and bootstrap admin.conf.

it feels to me we should keep this default group and let kubeadm init always use it - i.e. "this is the default kubeadm admin group, that we always create for you".
users can then add more groups and users with `kubectl ...` and `kubeadm kubeconfig ...`.

i talk about this in the kubeadm-certs.md update in this draft k/website PR:
https://github.com/kubernetes/website/pull/43540/files